tests/distributions/test_ordered_logistic.py

```diff
@@ -75,3 +80,48 @@ def test_autograd():
     assert torch.all(predictor.grad != 0).item()
     assert cutpoints.grad is not None
     assert torch.all(cutpoints.grad != 0).item()
+
+
+# Tests for the OrderedTransform
+
+
+@pytest.mark.parametrize("batch_shape", [(), (1,), (5,), (5, 5), (1, 5), (5, 1)])
+@pytest.mark.parametrize("event_shape", [(1,), (5,), (100,)])
+def test_transform_bijection(batch_shape, event_shape):
+    tf = OrderedTransform()
+    assert tf.inv.inv is tf
+    shape = torch.Size(batch_shape + event_shape)
+    sample = Normal(0, 1).expand(shape).sample()
+    tf_sample = tf(sample)
+    inv_tf_sample = tf.inv(tf_sample)
+    assert torch.allclose(sample, inv_tf_sample)
+
+
+def cjald(func, X):
+    """cjald = Computes Jacobian Along Last Dimension
+    Recursively splits tensor ``X`` along its leading dimensions until we are
+    left with a vector, computes the jacobian of this vector under the
+    transformation ``func``, then stitches all the results back together using
+    ``torch.stack``.
+    """
+    assert X.ndim >= 1
+    if X.ndim == 1:
+        return jacobian(func, X)
+    else:
+        return torch.stack([cjald(func, X[i]) for i in range(X.shape[0])], dim=0)
```

Comment:
nice implementation!

Reply:
thanks, I thought it was pretty slick!

Reply:
Indeed slick! You could even condense :smile:
```py
torch.stack([cjald(func, x) for x in X.unbind(0)], dim=0)
```